pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -991,6 +991,12 @@ func (a *ApplicationBackupController) backupResources(
 	var err error
 	var resourceTypes []metav1.APIResource
 	// Listing all resource types
+	if len(backup.Spec.ResourceTypes) != 0 && len(backup.Spec.IncludeResources) != 0 {
+		// Currently this is not supported, error out
+		errMsg := "cannot have ResourceType and IncludeResources together"
+		log.ApplicationBackupLog(backup).Errorf(errMsg)
+		return fmt.Errorf(errMsg)
+	}
```

Comment:
Do we need to have this check in backupVolume as well, where we access the includeResource for extracting PVCs

Reply:
Added in Handle